Parsers/ASimNetworkSession/ProductParsers/ASimNetworkSessionPaloAltoPanOS.yaml

```diff
@@ -0,0 +1,118 @@
+Parser:
+  Title: Palo Alto Network Sessions
+  Version: '0.1'
+  LastUpdated: Dec 20, 2021
+Product:
+  Name: PaloAlto
+Normalization:
+  Schema: NetworkSessions
+  Version: '0.2.0'
+References:
+- Title: ASIM Network Session Schema
+  Link: https://aka.ms/ASimNetworkSessionDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+Description: |
+  This Query Parser maps Palo Alto PanOS Firewall Events (CommonSecurityLogs) to the Advanced SIEM Information Model Network Session schema.
+ParserName: ASimNetworkSessionPaloAltoPanOS
```

Comment:
I think we should include CEF in the name, so that we can add a Palo Alto parser that is not based on CEF, for example with additional fields. What do you think?

Reply:
Makes sense, bu I think this parser name is too long as is (31 chars!). Maybe rename to "ASimNetworkSessionPanOSCEF".
Do you think we are going to have PAN = union isfuzzy (PAN_CeF, PAN_Syslog)